config/install/user.role.az_content_admin.yml

```diff
@@ -44,6 +57,8 @@ permissions:
   - 'use editorial transition archived_published'
   - 'use editorial transition create_new_draft'
   - 'use editorial transition publish'
+  - 'use text format full_html'
```

Comment:
Can we remove this from the PR for a separate discussion? We had initially left this permission out intentionally.

Reply:
Absolutely. I didn't intend to add that.

Reply:
Must have been added in another PR at some point. I have removed it for this PR.

Reply:
I turned the full html text editor permission off in my local build, but it still shows up in the probo build @camikazegreen. I don't actually see it in my local .yml file either. Is it coming from somewhere else?